Podfile

```diff
@@ -13,7 +14,7 @@ raise 'Please run CocoaPods via `bundle exec`' unless %w[BUNDLE_BIN_PATH BUNDLE_
 inhibit_all_warnings!
 use_frameworks!
 
-app_ios_deployment_target = Gem::Version.new('13.0')
+app_ios_deployment_target = Gem::Version.new(Xcodeproj::Config.new(File.new('./config/Common.xcconfig')).to_hash['IPHONEOS_DEPLOYMENT_TARGET'])
```

Comment:
```suggestion
PROJECT_ROOT_FOLDER = File.expand_path(__dir__)
COMMON_CONFIG_PATH = File.join(PROJECT_ROOT_FOLDER, 'config', 'Common.xcconfig')

app_ios_deployment_target = Gem::Version.new(Xcodeproj::Config.new(COMMON_CONFIG_PATH).to_hash['IPHONEOS_DEPLOYMENT_TARGET'])
```

Just a suggestion that handles pathing properly if `bundle exec pod` isn't run from the root directory.

Reply:
Nice. It didn't occur to me, thanks @jkmassel.

Indeed, the current version run from e.g. `fastlane` fails with file not found:

<img width="1261" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/2df64e11-0107-4bd9-bc51-e661aedb6d80">

Using an absolute path makes it robust:

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/369d23b6-8651-4299-9faf-604bccbab492)

---

I had trouble with absolute paths while working on the Gutenberg local podspec because the resulting hash was different across machines, thus breaking our `Podfile.lock` validation. But I don't think the `Podfile` checksum is done on the interpreted code. Let's see what CI says...